bin/ivar_variants_to_vcf.py

```diff
@@ -221,165 +242,159 @@ def ivar_variants_to_vcf(FileIn, FileOut, passOnly=False, minAF=0,NotStrandBias=
                     var_type = "DEL"
 
                 QUAL = "."
-                pass_test = line[13]
-                REF_CODON = line[15]
-                ALT_CODON = line[17]
 
-                if NotStrandBias:
-                    if pass_test =="TRUE":
+                ## Determine FILTER field
+                INFO = f"DP={line[11]}"
+                pass_test = line[13]
+                if ignore_strand_bias:
+                    if pass_test == "TRUE":
                         FILTER = "PASS"
                     else:
                         FILTER = "FAIL"
-                    INFO = "DP=" + line[11]
                 else:
-                    # If strand-bias test significative add SB in the FILTER field, else PASS
-                    if p<0.05 and pass_test =="TRUE":
+                    ## Add SB in the FILTER field if strand-bias p-value is significant
+                    if pvalue < 0.05 and pass_test == "TRUE":
                         FILTER = "SB"
-                    elif p>0.05 and pass_test =="TRUE":
+                    elif pvalue > 0.05 and pass_test == "TRUE":
                         FILTER = "PASS"
-                    elif  p<=0.05 and pass_test == "FALSE":
-                        FILTER = "SB,other"
+                    elif  pvalue <= 0.05 and pass_test == "FALSE":
+                        FILTER = "FAIL"
                     else:
                         FILTER = "FAIL"
-                    INFO = "DP=" + line[11]+":SB_pvalue="+str(round(p,5))
+                    INFO += f":SB={str(round(pvalue, 5))}"
```

Comment:
From Slack: https://nfcore.slack.com/archives/C0110R22NH3/p1643881900209169

Me: 
Should we make this `SB` instead of `SB_pvalue` ?

Me:
Does it matter having the same id in `INFO` and `FORMAT` fields? Not going to break anything downstream right?

Reply:
which ID is the same in INFO and Format? SB is in FILTER and INFO, right?

Reply:
Yep, will it matter if we use `SB` in both the `FILTER` and `INFO` sections? It won't break any downstream VCF validation right?

Reply:
I don't think so...we could use `SB_PV` in the `INFO` field though so it is clearer

Reply:
Added `SB_PV` in https://github.com/nf-core/viralrecon/pull/272/commits/94fa8ff2cad0d1aea4556f2dab5471874e79c4b2.